meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb

```diff
@@ -27,8 +26,11 @@ inherit deploy
 do_configure[noexec] = "1"
 
 do_compile() {
-	sed -e '/@@INCLUDE_COMMON_IMX@@/ {' -e 'r ${S}/boot-common-imx.cmd.in' -e 'd' -e '}' \
-			"${S}/boot.cmd" > boot.cmd.in
+	# Check if any modifications were done by bsp specific
+	# do_compile() prepends from meta-lmp-bsp
+	if [ ! -f boot.cmd.in ]; then
+		cp "${S}/boot.cmd" boot.cmd.in
```

Comment:
The comment is sort of backwards.  This handles a case where there is no BSP modification and we're using the original `boot.cmd`

Reply:
fixed

Reply:
perfect thanks!